pom.xml

```diff
@@ -21,13 +21,23 @@
         <dependency>
             <groupId>com.offbytwo.jenkins</groupId>
             <artifactId>jenkins-client</artifactId>
-            <version>0.3.6</version>
+            <version>0.3.7-SNAPSHOT</version>
```

Comment:
asked in mailing list: https://groups.google.com/forum/#!topic/java-client-api/boD3H2KCdFg